platform/windows/joypad_windows.h

```diff
@@ -85,6 +85,8 @@ class JoypadWindows {
 			last_pad = -1;
 			attached = false;
 			confirmed = false;
+			di_joy = nullptr;
+			guid = { 0x0000, 0x0000, 0x0000, { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00 } };
```

Comment:
I guess `guid = {};` isn't sufficient for this?

Reply:
`{}` seems to be good for GCC and Clang, so it's probably OK.